### ISSUE TITLE
Weird behavior for Style/IndentationWidth rule when assigning from an if statement

### ISSUE THREAD

**Issue by rylanc** [bbatsov/rubocop#2278]
I'm getting an unexpected warning when running the following code through version 0.34.2:
```ruby
foo = if something
  'one'
else
  'two'
end
```

Output:
```
test.rb:2:3: C: Use 2 (not -4) spaces for indentation.
  'one'
  ^^^^
```

When run with `--auto-correct`, it produces:
```ruby
bla = if something
        'one'
else
  'two'
end
```

**Comment by jonas054** [comment]
Use the following configuration to make RuboCop accept the code:
```yaml
Lint/EndAlignment:
  AlignWith: variable
```
The reason you don't get any auto-correction of the `else` is probably that you have disabled `Style/ElseAlignment` in your `.rubocop.yml` or `.rubocop_todo.yml`.

If you want auto-correction of the `end` keyword, you can add
```yaml
Lint/EndAlignment:
  AutoCorrect: true
```

**Comment by alexdowad** [comment]
Looks like this can be closed.
